main.py

```diff
@@ -1,19 +1,22 @@
 import eel
 import os
 from pytube import YouTube
+
 eel.init("web")
+
+
 @eel.expose
-def download_video(link):
+def download_video(link: str) -> str:
     print("Download Started")
     eel.set_loading_cursor()
     yt = YouTube(link)
     audio_stream = yt.streams.get_lowest_resolution()
-    if os.path.isdir("web/downloads/") == True:
```

Comment:
Here we dont need to pass, we can just create the else as the if statements condition...

Reply:
ah thanks